com.unity.ml-agents/Runtime/Sensor/CameraSensor.cs

```diff
@@ -102,5 +133,10 @@ public static Texture2D ObservationToTexture(Camera obsCamera, int width, int he
             RenderTexture.ReleaseTemporary(tempRt);
             return texture2D;
         }
+
+        internal static int[] GenerateShape(int width, int height, bool grayscale)
```

Comment:
Does this need a docstring?

Reply:
Doesn't for package validation - but I added one for completeness.